src/main/java/uk/nhs/hee/tis/trainee/credentials/service/GatewayService.java

```diff
@@ -100,12 +105,20 @@ private HttpEntity<MultiValueMap<String, String>> buildParRequest(CredentialDto
     MultiValueMap<String, String> bodyPair = new LinkedMultiValueMap<>();
     bodyPair.add("client_id", properties.clientId());
     bodyPair.add("client_secret", properties.clientSecret());
-    bodyPair.add("redirect_uri", properties.issuing().redirectUri());
     bodyPair.add("scope", dto.getScope());
     bodyPair.add("id_token_hint", idTokenHint);
     bodyPair.add("nonce", nonce);
     bodyPair.add("state", state);
 
+    if (properties.issuing().callbackUri() != null) {
+      bodyPair.add(REDIRECT_URI, properties.issuing().callbackUri()); //callback for logging
+    } else {
+      bodyPair.add(REDIRECT_URI, properties.issuing().redirectUri());
+    }
```

Comment:
What's the difference between `callbackUri` and `redirectUri`?
Would we ever not want to hit the callback endpoint?

Reply:
The `callbackUri` is the endpoint for logging the issued credential (which then redirects to the `redirectUri`, which is probably a front-end page). Skipping the `callbackUri` would mean that there was no logging, but I can't think of a use-case where we'd want to enable that (or have it accidentally enabled by omitting a parameter value for `callbackUri`). I'll remove the logic for now.